cypress/integration/opensearch-dashboards/without-security/dashboard_sanity_test_spec.js

```diff
@@ -26,6 +26,8 @@ describe('dashboard sample data validation', () => {
         before(() => {
             // Go to the home page
             miscUtils.visitPage('app/home#')
+            cy.window().then(win => win.localStorage.setItem('home:welcome:show', false))
```

Comment:
Nice fix!

Can you also undo this change in the `after()` function? This is to ensure every test case is cleaning up its own data.

Reply:
add the remove in after section